polymer/slambook/lib/tute_slambookform.html

```diff
@@ -0,0 +1,98 @@
+<!--
+  Copyright (c) 2012, the Dart project authors.  Please see the AUTHORS file
+  for details. All rights reserved. Use of this source code is governed by a
+  BSD-style license that can be found in the LICENSE file.
+-->
+
+<dom-module id="tute-slambookform" extends="form">
+
+ <style>
```

Comment:
One note, as of Polymer JS 1.1 they now allow the style tags inside the template tag again, LOL. Since it means one less difference from the old version we should probably do it that way (both are supported though)